.gcloudignore

```diff
@@ -0,0 +1,11 @@
+.gcloudignore
+.gitignore
+LICENSE
+tsconfig.json
+tslint.json
+yarn.lock
+
+.git/
+/.config
+/node_modules
```

Comment:
We should do `node_modules/` so that if any nested `node_modules` will be generated, these won't be pushed accidentally?